### PR TITLE
Remove jquery from base_update template

### PR DESCRIPTION
**Pull request by anselmbradford**
## Removals

- Removes jquery from base_update template. 

## How to test this PR

1. This template appears to only be used on:
http://localhost:8000/data-research/consumer-complaints/
https://www.consumerfinance.gov/paying-for-college2/understanding-your-financial-aid-offer/offer/test/

### CODE REVIEW

**Review by wpears — CHANGES_REQUESTED** [review]
The `nemo_js` appears to also be used on:
`cfgov/paying_for_college/templates/paying_for_college/disclosure_technote.html`
(though doesn't look like any jquery is actually invoked there)
and there's a reference to the `nemo_js` block on:
`cfgov/legacy/templates/students/helping-borrowers-find-ways-to-stay-afloat/index.html`
but the parent/child relationship is less clear to me on that page.

**Comment by anselmbradford** [comment]
Thanks @wpears !

> The nemo_js appears to also be used on:
cfgov/paying_for_college/templates/paying_for_college/disclosure_technote.html
(though doesn't look like any jquery is actually invoked there)

Removed in https://github.com/cfpb/consumerfinance.gov/pull/6031/commits/38f9beeea5ce0a91b9482b3fa624db692fa47ae8

> and there's a reference to the nemo_js block on:
cfgov/legacy/templates/students/helping-borrowers-find-ways-to-stay-afloat/index.html
but the parent/child relationship is less clear to me on that page.

This uses a different template and is removed in https://github.com/cfpb/consumerfinance.gov/pull/6030

**Review by wpears — APPROVED** [review]
:+1:

File: cfgov/legacy/templates/front/base_update.html [hunk 1]
```diff
@@ -211,16 +204,6 @@
...
-<script type="text/javascript">
-    // If there are no breadcrumbs on this page, remove the ul tag so that screen
-    // readers don't read it (since the ul has nothing in it).
-    $(function(){
-        var numBreadCrumbs = $( 'ul.bread' ).children( 'li' ).length;
-        if( numBreadCrumbs === 0 ) {
-           $( 'ul.bread' ).remove();
-        }
-    });
-</script>
```
Comment (Scotchester) [inline cfgov/legacy/templates/front/base_update.html:223]: You can also remove the `<ul class="bread meta">` from this template, as nothing that extends this template puts anything in `{% block breadcrumbs %}`.

**Review by Scotchester — APPROVED** [review]
One small change that could be thrown in, but otherwise looks good.

**Pull request merged**